AdminWebsite/AdminWebsite/Mappers/AddNotificationRequestMapper.cs

```diff
@@ -316,21 +316,27 @@ public static class AddNotificationRequestMapper
             {
                 notificationType = NotificationType.StaffMemberDemoOrTest;
                 parameters.Add(NotifyParams.StaffMember, $"{participant.FirstName} {participant.LastName}");
-            }
+            }            
             else if (participant.UserRoleName.Contains(RoleNames.Judge, StringComparison.InvariantCultureIgnoreCase))
             {
-                if (hearing.IsJudgeEmailEJud())
-                {
-                    notificationType = NotificationType.EJudJudgeDemoOrTest;
-                }
-                else
-                {
-                    notificationType = NotificationType.JudgeDemoOrTest;
-                    parameters.Add(NotifyParams.CourtroomAccountUserName, participant.Username);
-                }
+                var judgeEmail = hearing.GetJudgeEmail();
+                    if (string.IsNullOrEmpty(judgeEmail))
```

Comment:
Do we not want to use the email for the judge account as a fall back? It doesn't seem clear from the ticket so I need clarification on that.

Also it looks like we don't show the judge email field for ejud users so ejud judges will never get an email.